src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
-import './MoviesList.scss';
+// import './MoviesList.scss';
+import '../../App.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    <ul>
+      {movies.map((movie) => (
+        <li key={movie.imdbId}>
```

Comment:
`<li>` tag should be inside `MovieCard` component instead of `<div>` with `className="card"`

Reply:
Should we separate `<ul>` parent element with child `<li>`?